columnq/src/table/mod.rs

```diff
@@ -108,12 +109,14 @@ impl TableOptionCsv {
     }
 
     #[inline]
+    #[must_use]
     pub fn with_delimiter(mut self, d: u8) -> Self {
         self.delimiter = d;
         self
     }
 
     #[inline]
+    #[must_use]
```

Comment:
could you help provide some contexts on what's the reason for adding these must_use attributes?